RecoEgamma/EgammaElectronProducers/plugins/GEDGsfElectronCoreProducer.cc

```diff
@@ -1,56 +1,61 @@
-
-#include "GEDGsfElectronCoreProducer.h"
-
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
 #include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
-
 #include "DataFormats/EgammaCandidates/interface/GsfElectronCore.h"
-#include "DataFormats/ParticleFlowReco/interface/GsfPFRecTrack.h"
-#include "DataFormats/GsfTrackReco/interface/GsfTrack.h"
```

Comment:
Again, correct me if I'm wrong, but I think these `*Fwd.h` are not meant to be included in `*.cc` files, and that's why I removed them. Indeed, I'm relying here on the indirect include via `GsfElectronCore.h` which I think is fine because the *Track objects in this producer are only used as arguments for `GsfElectronCore` constructors or member functions, so we can rely on `GsfElectronCore.h` to have the good typedefs imported to use it's own functions. Is that OK for you?